en/guide/using-template-engines.md

```diff
@@ -12,28 +12,28 @@ A _template engine_ enables you to use static template files in your application
 variables in a template file with actual values, and transforms the template into an HTML file sent to the client.
 This approach makes it easier to design an HTML page.
 
-Some popular template engines that work with Express are [Jade](http://jade-lang.com/),
+Some popular template engines that work with Express are [Pug](http://jade-lang.com/),
 [Mustache](https://www.npmjs.com/package/mustache), and [EJS](https://www.npmjs.com/package/ejs).
-The [Express application generator](/{{ page.lang }}/starter/generator.html) uses Jade as its default, but it also supports several others.
+The [Express application generator](/{{ page.lang }}/starter/generator.html) uses Pug as its default, but it also supports several others.
 
 See [Template Engines (Express wiki)](https://github.com/strongloop/express/wiki#template-engines)
 for a list of template engines you can use with Express.
-See also [Comparing JavaScript Templating Engines: Jade, Mustache, Dust and More](https://strongloop.com/strongblog/compare-javascript-templates-jade-mustache-dust/).
+See also [Comparing JavaScript Templating Engines: Pug, Mustache, Dust and More](https://strongloop.com/strongblog/compare-javascript-templates-jade-mustache-dust/).
```

Comment:
This needs to be reverted (for all languages) because the old blog post hasn't changed.